benchmark/fs/bench-chownSync.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+const common = require('../common');
+const fs = require('fs');
+const assert = require('assert');
+const tmpdir = require('../../test/common/tmpdir');
+
+if (process.platform === 'win32') {
+  // Windows does not have `getuid` or `getgid`
+  process.exit(0);
+}
```

Comment:
The issue on benchmarks is related to this condition, you should print at least one line using `process.stdout.write`, otherwise, the test will fail.

I sent you a suggestion on twitter on how to deal with this condition.